tensorflow/core/BUILD

```diff
@@ -1762,6 +1762,9 @@ tf_cc_tests(
         "graph/graph_test.cc",
         "graph/node_builder_test.cc",
         "graph/optimizer_cse_test.cc",
+# INTEL_MKL start
```

Comment:
yes this is what I did, but if_mkl() itself has a select function and it complains because of that:

def if_mkl(if_true, if_false = []):
    return select({
        "//third_party/mkl:using_mkl": if_true,
        "//conditions:default": if_false
    })